File: swagger/models/get_memberships_request.py
from .base_model import BaseModel


class GetMembershipsRequest(BaseModel):
	"""
	no description available

	Attributes:

		membership_ids
		Filters results to memberships that belong to one of the given
		membership IDs. If omitted, all memberships are returned.
	"""

	input_map = {
		'MembershipIds': 'membership_ids',
		}

	output_map = {
		'membership_ids': 'MembershipIds',
		}

File: swagger/Models/GetMembershipsRequest.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetMembershipsRequest extends BaseModel
{
	/**
	 * Filters results to memberships that belong to one of the given
	 * membership IDs. If omitted, all memberships are returned.
	 * 
	 * @var integer[] 
	 */
	public $membershipIds;

	protected function getInputMap()
	{
		return [
			'MembershipIds' => ['membershipIds', 'array', null],
			];
	}

	protected function getOutputMap()
	{
		return [
			'membershipIds' => ['MembershipIds', 'array', null],
			];
	}
}
